frontend/src/components/layouts/SettingsLayout.tsx

```diff
@@ -60,7 +61,7 @@ export default function SettingsLayout() {
     <>
       <AppHeader
         title="Settings"
-        description="Manage your Alby Hub settings."
+        description=""
```

Comment:
@stackingsaunter is this intentional? it seems inconsistent with the rest of the pages

Reply:
it was as per designs. readded it to be consistent